staking_deposit/utils/constants.py

```diff
@@ -13,7 +13,7 @@
 ETH1_ADDRESS_WITHDRAWAL_PREFIX = bytes.fromhex('01')
 
 ETH2GWEI = 10 ** 9
-MIN_DEPOSIT_AMOUNT = 2 ** 0 * ETH2GWEI
+MIN_DEPOSIT_AMOUNT = 1 ** 0 * ETH2GWEI
```

Comment:
this one is actually ok imo. both 2 ** 0 and 1 ** 0 are 1, right?

Reply:
Updated